### ISSUE TITLE
Iron Hands Head of the Gorgon

### ISSUE THREAD

**Issue by LachrimallusExterminator** [BSData/horus-heresy#2028]
I have been holding off reporting these for a long time. A Forge Lord Consul should not count towards the 0-1 restriction of consuls for this Rite of War.

![93D16B08-69AE-4D93-AAD5-AF41020D74D9](https://user-images.githubusercontent.com/84422145/151665941-b73ffaad-fce9-406a-8031-6c76c182a86a.png)
![96720C0B-B775-4E49-B747-73BA96B451BC](https://user-images.githubusercontent.com/84422145/151665944-ad86f9e1-db52-4088-9733-5effe2b06a84.png)

Previously the Praevian’s Cortex Controller allowing the selection of and addition Mechanicum unit. This no longer works as the Preavian’s own unit consumes it. Is this deliberate?

![0E442788-7BDD-455B-9F83-AC4A147712F7](https://user-images.githubusercontent.com/84422145/151666056-c238b0ba-2919-4b6a-aad9-b95325e5658c.png)

 - OS: iOS 14.8.1
 - BattleScribe version 2.03.15
 - Release version v674

**Comment by Mayegelt** [comment]
Never hold off. It's an easy fix, but 8 hadn't fixed it as I didn't know it was broken.

**Comment by Mayegelt** [comment]
The Pravian part I'll look at. I need to check some wording. It's probably not a deliberate change, but something that has happened by accident.
It should be an easy fix by just adding an additional cortex controller catagory to the pravian somewhere.

**Comment by Mayegelt** [comment]
Fixed my side, will be in the next update. Though discovered that the min 1 cortex controlled model per 3 controllers is broken in brethren of iron. So will need to fix that.

**Comment by LachrimallusExterminator** [comment]
> 

You mean min 1 controller per 3 controlled models?

**Comment by Mayegelt** [comment]
> > 
> 
> You mean min 1 controller per 3 controlled units?

Yeah. That is what I ment. All the fun of pain crossing wires in my brain.
